platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/StatusApplicationCommand.java

```diff
@@ -51,44 +51,45 @@ protected void doExecute(ApplicationService applicationService)
       throws ApplicationServiceException {
 
     Application application = applicationService.getApplication(appName);
-    if (application != null) {
-      ApplicationStatus appStatus = applicationService.getApplicationStatus(application);
-      console.println(application.getName());
-      console.println("\nCurrent State is: " + appStatus.getState().toString());
 
-      console.println("\nFeatures Located within this Application:");
-      for (Feature curFeature : application.getFeatures()) {
-        console.println("\t" + curFeature.getName());
-      }
+    if (application == null) {
+      console.println("No application found with name " + appName);
+      return;
+    }
+
+    ApplicationStatus appStatus = applicationService.getApplicationStatus(application);
+    console.println(application.getName());
+    console.println("\nCurrent State is: " + appStatus.getState());
```

Comment:
❓ Any particular reason for the `\n` here since using `println`? Formatting?

Reply:
^ Probably just formatting